src/item.cpp

```diff
@@ -306,12 +248,45 @@ bool item::covers( const body_part bp ) const
         // go on another bodypart.
         return bp == bp_torso;
     }
-    return covered_bodyparts.test( bp );
+    return get_covered_body_parts().test(bp);
 }
 
-const std::bitset<num_bp> &item::get_covered_body_parts() const
+const std::bitset<num_bp> item::get_covered_body_parts() const
```

Comment:
Please remove the `const` on the bitset (here and in the declaration in item.h), it's not needed now as the function returns a value and not a reference.